bin/setup

```diff
@@ -11,6 +11,7 @@ def setup!
     copy "config/database.example.yml"
     copy "config/secrets.example.yml"
     test_local_env_contains_required_keys
+    run  "bundle exec rails generate strong_migrations:install"
```

Comment:
I've done it this way because the config file that it creates automatically pulls in the time that it's run. Theoretically could just copy in the initializer that this generates, but this way is more dynamic.